pkg/flowaggregator/kafka/convertor.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2021 VMware, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafka
+
+import (
+	"net"
+
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/kafka/producer/convertor"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/flowaggregator/kafka/protobuf"
+)
+
+const (
+	AntreaFlowMsg string = "AntreaFlowMsg"
+	AntreaTopic   string = "AntreaTopic"
+)
```

Comment:
this doesn't make it clear what the constant represents when you use it

if possible we should have somthing like this

```go
type ProtobufSchema string
const (
    ProtobufSchemaAntreaFlowMsg ProtobufSchema = "AntreaFlowMsg"
)
```